src/Cmfd.cpp

```diff
@@ -2637,11 +2637,10 @@ CMFD_PRECISION Cmfd::getFluxRatio(int cell_id, int group, int fsr) {
 
   double* interpolants;
   double ratio = 1.0;
-  if (_k_nearest > 0)
+  if (_use_axial_interpolation)
     interpolants = _axial_interpolants.at(fsr);
-
-  if (_k_nearest > 0 and (interpolants[0] != 0 or interpolants[2] != 0)) {
-
+  if (_use_axial_interpolation && _local_num_z >= 3 && 
+    ( interpolants[0] != 0 || interpolants[2] != 0) ) {
```

Comment:
(interpolants[0] != 0 || interpolants[2] != 0))

Reply:
Do you mean avoiding comparing the equality of float?

Reply:
No it's a style comment. Parenthesis don't need an extra whitespace inside